javav2/README.rst

```diff
@@ -26,9 +26,9 @@ To build and run these examples, you need the following:
   
   After you set your AWS credentails in the credentials file located in the .aws folder, you can create a Service Client like this.

```

Comment:
"in the credentials file located in the .aws folder" -- for what it's worth, there are other ways to set AWS credentials.

Reply:
Addressed this comment